service/src/util/baseUtils.ts

```diff
@@ -14,6 +14,56 @@ export type DetailedEntry = fhir4.BundleEntry & {
   outcome?: OperationOutcome;
 };
 
+/**
+ * Add isOwned extension to the main Library reference on a Measure's relatedArtifacts
```

Comment:
```suggestion
 * Checks entry for Measure type with library and adds isOwned extension to the main Library reference on a Measure's relatedArtifacts
```
Thought it might be helpful to note that this can be applied to any entry because it can do the checking whether the reference should be added. Feel free to change wording though!